pmg/views.py

```diff
@@ -230,11 +232,17 @@ def bills(bill_type, year=None):
             abort(404)
         params['filter[year]'] = year
 
-    api_url = 'bill' if bill_type == 'all' else 'bill/%s' % bill_type
-    bills = load_from_api(
-        api_url, return_everything=True, params=params)['results']
+    api_url = "bill" if bill_type == "all" else "bill/%s" % bill_type
+    bills = load_from_api(api_url, return_everything=True, params=params)["results"]
+
+    if bill_type == "pmb":
+        api_url = "bill/committee"
+        committee = load_from_api(api_url, return_everything=True, params=params)[
+            "results"
+        ]
+        bills = bills + committee
 
-    bills.sort(key=lambda b: [-b['year'], b['code'][0],  b.get('number', 0), b['title']])
+    bills.sort(key=lambda b: [-b["year"], b["code"][0], b.get("number", 0), b["title"]])
 
```

Comment:
- it's not really bill_type, more like bill_view
- keeping the web URL as /bills/pmb but showing pmb and committee bills is confusing but not the end of the world
- formatting in `pmb` for the first lookup and tagging `committee` bills on as a special case just compounds all the issues into something that goes over my threshold what's readable code.

I think fixing the slug is sufficient, because it forces the bill lookup for the "pmb and committees" case to be more explicit.

Here's how I would do it

```python
@app.route('/bills/<any(current):bills_view>/')
@app.route('/bills/<any(all, draft, pmb, tabled):bills_view>/') # redirect to year version - next line
@app.route('/bills/<any(all, draft, pmb, tabled):bills_view>/year/<int:year>/')
def bills(bills_view, year=None):
    if bills_view == 'current':
        # don't paginate by year
        year_list = None
        params = {}

    else:
        year_list = range(MIN_YEAR, date.today().year + 1)
        year_list.reverse()
        params = {}

        if not year:
            return redirect(
                url_for('bills', bills_view=bills_view, year=year_list[0]))

        if year not in year_list:
            abort(404)
        params['filter[year]'] = year

    if bills_view == 'all':
        api_url = 'bill'
        bills = load_from_api(
            api_url, return_everything=True, params=params)['results']
    elif bills_view == 'pmb-commitee':
        api_url = 'bill/pmb'
        pmb = load_from_api(
            api_url, return_everything=True, params=params)['results']
        api_url = 'bill/committee'
        committee = load_from_api(
            api_url, return_everything=True, params=params)['results']
        # merge
        bills = pmb + committee
    else:  # current, draft, tabled
        api_url = 'bill/%s' % bills_view
        bills = load_from_api(
            api_url, return_everything=True, params=params)['results']
```